Example/NVActivityIndicatorViewExample/ViewController.swift

```diff
@@ -41,30 +41,32 @@ class ViewController: UIViewController, NVActivityIndicatorViewable {
         let cellWidth = Int(self.view.frame.width / CGFloat(cols))
         let cellHeight = Int(self.view.frame.height / CGFloat(rows))
 
-        (NVActivityIndicatorType.ballPulse.rawValue ... NVActivityIndicatorType.circleStrokeSpin.rawValue).forEach {
-            let x = ($0 - 1) % cols * cellWidth
-            let y = ($0 - 1) / cols * cellHeight
+        let presentingIndicatorTypes = NVActivityIndicatorType.allCases.filter { $0 != .blank }
```

Comment:
I'd suggest refactor this as a VC member to avoid repetition.

Reply:
Good idea. Done.